### ISSUE TITLE
Update doc for twistd

### ISSUE THREAD

**Issue by AdrienDS** [twisted/klein#203]
The example in the [doc](http://klein.readthedocs.io/en/latest/examples/twistd.html) works well with Twisted v16, but not with Twisted v17

With `klein==17.2.0`

```
$ pip install Twisted==16.2.0
$ twistd -n web --class=helloworld.resource
[twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 16.2.0 ([...]/Python 2.7.13) starting up.
[twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.selectreactor.SelectReactor.
[-] Site starting on 8080
[twisted.web.server.Site#info] Starting factory <twisted.web.server.Site instance at 0x103e640e0>
```

```
$ pip install Twisted==17.5.0
$ twistd -n web --class=helloworld.resource
Traceback (most recent call last):
  File "[...]/2.7/bin/twistd", line 11, in <module>
    load_entry_point('Twisted==17.5.0', 'console_scripts', 'twistd')()
  File "[...]/2.7/lib/python2.7/site-packages/twisted/scripts/twistd.py", line 29, in run
    app.run(runApp, ServerOptions)
  File "[...]/2.7/lib/python2.7/site-packages/twisted/application/app.py", line 657, in run
    config.parseOptions()
  File "[...]/2.7/lib/python2.7/site-packages/twisted/application/app.py", line 624, in parseOptions
    usage.Options.parseOptions(self, options)
  File "[...]/2.7/lib/python2.7/site-packages/twisted/python/usage.py", line 267, in parseOptions
    self.subOptions.parseOptions(rest)
  File "[...]/2.7/lib/python2.7/site-packages/twisted/python/usage.py", line 255, in parseOptions
    self._dispatch[optMangled](optMangled, arg)
  File "[...]/2.7/lib/python2.7/site-packages/twisted/python/usage.py", line 411, in <lambda>
    fn = lambda name, value, m=method: m(value)
  File "[...]/2.7/lib/python2.7/site-packages/twisted/web/tap.py", line 120, in opt_class
    classObj = reflect.namedClass(className)
  File "[...]/2.7/lib/python2.7/site-packages/twisted/python/reflect.py", line 166, in namedObject
    module = namedModule('.'.join(classSplit[:-1]))
  File "[...]/2.7/lib/python2.7/site-packages/twisted/python/reflect.py", line 152, in namedModule
    topLevel = __import__(name)
ImportError: No module named helloworld
```

**Comment by glyph** [comment]
This is an artifact of switching to entrypoints, which removes `.` from sys.path when using `twist`, `twistd`, and `trial`.  If you do `PYTHONPATH=. twistd -n web --class=helloworld.resource` it should work great, or if you make a `setup.py` to install helloworld.

To be clear, this is definitely a bug in the docs, it needs to be fixed; just leaving the solution here for anyone who wants to do the fixing.

**Comment by wsanchez** [comment]
I wonder if `--class` should handle explicitly relative names, like `import`… meaning this could work `--class=.helloworld.resource` ?

**Comment by AdrienDS** [comment]
Thanks, I opened a pull request with @glyph's solution with `PYTHONPATH=.`

**Comment by wsanchez** [comment]
Fixed by #204
